src/main/java/seedu/duke/command/DeleteEventCommand.java

```diff
@@ -28,6 +27,8 @@ public static String getCommandUsage() {
     }
 
     private static final String COMMAND_SUCCESSFUL_MESSAGE = "Event deleted:" + InterfaceManager.LS;
+    private static final String COMMAND_UNSUCCESSFUL_MESSAGE = "Event failed to delete: " + InterfaceManager.LS;
+    private static final String INDEX_OUT_OF_RANGE = "Index is out of range.";
```

Comment:
To standardize with your other PR
```suggestion
    private static final String INDEX_OUT_OF_RANGE_MESSAGE = "Index is out of range.";
```